src/debugger/SymbolManager.hh

```diff
@@ -19,11 +19,12 @@ class CommandController;
 
 struct Symbol
 {
-	Symbol(std::string n, uint16_t v)
-		: name(std::move(n)), value(v) {} // clang-15 workaround
+	Symbol(std::string n, uint16_t v, int16_t s)
+		: name(std::move(n)), value(v), slots(s) {} // clang-15 workaround
 
 	std::string name;
 	uint16_t value;
+	int16_t slots;
```

Comment:
I know the old debugger worked like this. But is it really useful to be able to select an arbitrary set of (sub)slots? Wouldn't it be sufficient to be able to select:
* no slots (this symbol is not tied to memory).
* all slots (this symbol is valid in all slots)  (and actually do we need both no slots and all slots?).
* **one** specific slot.

I can't immediately think of a situation where one symbol applies to more than one slot, but not to all slots.